index.html

```diff
@@ -183,35 +183,31 @@ <h3 id="unacceptablebehaviour">
+  <li><strong>Appreciate and Accommodate Our Similarities and Differences</strong>We come from many cultures and backgrounds, ways of life, and standard of behavior. Cultural differences can encompass everything from official religious observances to personal habits to clothing. Be respectful of people with different practices, attitudes, and beliefs. To help us achieve and maintain these high standards, each individual participant is expected to share responsibility for our work environment by adhering to the following behavioral guidelines:
+</li>
 	
-<p>To help us achieve and maintain these high standards, each individual participant is expected to share
-	responsibility for our work environment by adhering to the following behavioral guidelines:</p>
 	
-	<ol>
-		<li><strong>Respect.</strong> We are a large community of people who are passionate about our work, sometimes holding strong opinions beliefs. We are committed to dealing with each other with courtesy, respect, and dignity at all times. Misunderstandings and disagreements do happen. When conflicts arise, we are expected to resolve them maintaining that courtesy, respect, and dignity, even when emotions are heightened.</li>
+	<ul>
+		<li><strong>Respect.</strong> We are a large community of people who are passionate about our work, sometimes holding strong opinions and beliefs. We are committed to dealing with each other with courtesy, respect, and dignity at all times. Misunderstandings and disagreements do happen. When conflicts arise, we are expected to resolve them maintaining that courtesy, respect, and dignity, even when emotions are heightened.</li>
 		
-  <li>Never <a>harass</a> or <a>bully</a> anyone verbally, physically or 
-   <a data-lt="sexual harassment">sexually</a>.</li>
-  <li>Never <a>discriminate</a> on the basis of personal characteristics or 
-    group membership.</li>
-  <li>Communicate constructively and avoid <a>demeaning</a> or <a>insulting</a>
-    behavior or language.</li>
-  <li>Seek, accept, and offer objective work criticism, and <a>acknowledge</a> 
-    properly the contributions of others. </li>
-  <li>Be honest about your own qualifications, and about any circumstances that
```

Comment:
This sentence is removed and not replaced. haven't seen any documented consensus to remove it, or motivation for doing so.

My own view is that any requirement to document a conflict of interest belongs in the [Process](https://www.w3.org/2019/Process-20190301/#coi), not here, so it is fine to remove that part of the sentence.

Likewise my view is we don't need the part about being honest about your own qualifications - I'm not even sure what it means, in the sense that a "qualification" may be an official document that has little correlation with current knowledge or skill levels, or it may be an undocumented feature of experience. In either case I don't think we have a useful language for talking about qualifications that belongs in this document.

Reply:
We agree it should be removed.